infrastructure-aws-development.yml

```diff
@@ -0,0 +1,13 @@
+instance_groups:
+- name: master
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
+- name: minion
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
```

Comment:
After I fixed the cluster tag, I restarted (twice) and recreated (once) the nodes and didn't get stuck in `ContainerCreating` again. I'll add more services and recreate again for fun.